From bbe26c1596327936ee54bf2cd778bdb27203a9e6 Mon Sep 17 00:00:00 2001
From: Anil singh Bisht <Abisht@BHAVNACORP.NET>
Date: Thu, 4 Jul 2019 14:39:53 +0530
Subject: [PATCH] modify first

---
 src/first.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/first.txt b/src/first.txt
index 72f22fd..fd49d41 100644
--- a/src/first.txt
+++ b/src/first.txt
@@ -1,3 +1,5 @@
 first
 second
-t
\ No newline at end of file
+t
+f
+s
\ No newline at end of file
-- 
2.21.0.windows.1

