From 4ac4c3ac9863c41d5627bf449ad419a186547ae9 Mon Sep 17 00:00:00 2001
From: Bishta72-Hash <52492020+Bishta72-Hash@users.noreply.github.com>
Date: Wed, 3 Jul 2019 15:58:39 +0530
Subject: [PATCH] Initial commit

---
 README.md | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..5900e24
--- /dev/null
+++ b/README.md
@@ -0,0 +1,2 @@
+# test
+for learning only
-- 
2.21.0.windows.1

